### ISSUE TITLE
Editing fields is only available alternatively when we click 'Edit Configuration'

### ISSUE THREAD

**Issue by Arpit-yb** [yugabyte/yugabyte-db#8809]
**Description**
On the 2.6 portal , the edit configuration button under cloud provider config makes editing available alternatively.

**Steps to repro**
1. log in 2.6 portal
2. Go to Configs -> Backup -> amazon S3
3. Click on edit Configuration 
4. Click on Update without making any changes
5. Click on Edit configuration again
6. The field are not editable
<img width="1792" alt="Screenshot 2021-06-08 at 10 37 17 AM" src="https://user-images.githubusercontent.com/54769894/121126665-29d4a600-c846-11eb-90a9-f8fb28c78f5a.png">

**Comment by VijiYB** [comment]
cc @streddy-yb @ymahajan

**Comment by hsiaosu-yb** [comment]
Seens to work fine on portal right now.  Please double check the latest version to see if this bug reproduces.

**Comment by Arpit-yb** [comment]
I can still reproduce this issue in 2.6 portal

**Comment by nishantSharma459** [comment]
The same needs to be done in **`2.4`** as well.

**Comment by streddy-yb** [comment]
hi @Arpit-yb - does this repro in 2.4 as well?

**Comment by Arpit-yb** [comment]
Yes , i just checked and i am able to repro this bug in 2.4 as well
Link - http://portal2.4.dev.yugabyte.com/releases

**Comment by nishantSharma459** [comment]
@streddy-yb  Will take a look at 2.4 as well and raise a PR by EOD.

**Comment by Arpit-yb** [comment]
**Validated in the edit backup configuration in following scenarios**

1. Created a backup
2. Edited the backup 
3. Clicked Update(backup gets updated successfully)
4. Clicked edit config
5. Clicked Cancel(Previous state is retained)
6. Clicked Edit again
7.  Fields are editable

Works correctly
